perfkitbenchmarker/disk.py

```diff
@@ -73,8 +162,8 @@ def __init__(self, disk_size=None, disk_type=None,
 
   def ApplyFlags(self, flags):
     """Applies flags to the DiskSpec."""
-    self.disk_size = flags.scratch_disk_size or self.disk_size
-    self.disk_type = flags.scratch_disk_type or self.disk_type
+    self.disk_size = flags.data_disk_size or self.disk_size
+    self.disk_type = flags.data_disk_type or self.disk_type
     self.num_striped_disks = flags.num_striped_disks or self.num_striped_disks
```

Comment:
I added a comment covering your points 1 and 2. I would be happy to rename this flag -- I already have the flag-renaming machinery, so it's very easy to rename more flags -- but it wasn't in the design doc, so I think we'd need to have another round of discussion for this. What would you like to do?